packetnetworking/distros/debian/individual.py

```diff
@@ -4,15 +4,18 @@
 
 class DebianIndividualNetwork(NetworkBuilder):
     def build(self):
-        if self.network.bonding.link_aggregation == "individual":
-            self.build_tasks()
+        if self.network.bonding.link_aggregation != "individual":
+            return
+
+        super().build()
+        self.build_tasks()
 
     def build_tasks(self):
-        self.tasks = {}
+        template = "individual/etc_network_interfaces.j2"
+        if self.dhcp:
+            template = "dhcp/etc_network_interfaces.j2"
 
-        self.task_template(
-            "etc/network/interfaces", "individual/etc_network_interfaces.j2"
-        )
+        self.task_template("etc/network/interfaces", template)
```

Comment:
@mmlb For Ubuntu 20.04 and 18.04, shouldn't we create the network plans under `/etc/netplan`? That's how we get the Fortress-enabled servers working with the SmartNic.

Reply:
@shadowcolossus192 I don't think we need that